src/transformers/models/code_llama/tokenization_code_llama_fast.py

```diff
@@ -124,13 +124,18 @@ def __init__(
         suffix_token="▁<SUF>",
         eot_token="▁<EOT>",
         fill_token="<FILL_ME>",
+        additional_special_tokens=None,
         add_bos_token=True,
         add_eos_token=False,
+        use_default_system_prompt=False,
```

Comment:
The additional args should be documented in the doc string

Reply:
Indeed thanks

Reply:
(not for this PR) The `add_bos_token` and `add_eos_token` are not documented, and the args are in a very different order than the docstring